packages/venia-ui/lib/RootComponents/CMS/cms.js

```diff
@@ -1,8 +1,42 @@
-import React, { Component } from 'react';
-import CategoryList from '../../components/CategoryList';
+import React, { useEffect } from 'react';
+import { useQuery } from '@magento/peregrine';
+import cmsPageQuery from '../../queries/getCmsPage.graphql';
+import { fullPageLoadingIndicator } from '../../components/LoadingIndicator';
+import RichContent from '../../components/RichContent';
 
-export default class CMS extends Component {
-    render() {
-        return <CategoryList title="Shop by category" id={2} />;
+const CMSPage = props => {
+    const { id } = props;
+    const [queryResult, queryApi] = useQuery(cmsPageQuery);
+    const { data, error, loading } = queryResult;
+    const { runQuery, setLoading } = queryApi;
+
+    useEffect(() => {
+        setLoading(true);
+        runQuery({
+            variables: {
+                id: Number(id),
+                onServer: false
+            }
+        });
+    }, [id, runQuery, setLoading]);
+
+    if (error) {
+        return <div>Page Fetch Error</div>;
+    }
+
+    if (loading) {
+        return fullPageLoadingIndicator;
     }
-}
+
+    if (data) {
+        return (
+            <div>
+                <RichContent html={data.cmsPage.content} />
+                {/*<CategoryList title="Shop by category" id={2} />*/}
```

Comment:
What to do with this?  Commented for development but its original placement in here is strange

Reply:
RichContent should remove this, I imagine this was placed here due to the lack of other content on this page.

Reply:
Removed